.rubocop.yml

```diff
@@ -14,6 +14,8 @@ AllCops:
     - 'config/initializers/nationalities.rb'
     - 'lib/tasks/cucumber.rake'
     - 'features/support/env.rb'
+    - 'spec/support_specs/clockwork_spec.rb'
+    - 'spec/support/test_helpers/capture_stdout_helper.rb'
```

Comment:
rubocop complains about using $stdout directly, instead of using the rspec matcher .output(...).to_stdout -- using the rspec matcher is not appropriate here because we need to capture STDOUT output and process it (e.g. JSON.parse)

Reply:
the other complaint is about ran_job?, suggesting be_ran_job instead. I don't like how 'be_ran_job' reads, and the clockwork/test gem does not support it, anyway.